.laminas-ci.json

```diff
@@ -5,7 +5,8 @@
         "intl"
     ],
     "ignore_php_platform_requirements": {
-        "8.3": false
+        "8.3": false,
+        "8.4": false
```

Comment:
8.4 will need to be `true` here. Very few dependencies will be installable on 8.4

Reply:
Done